src/bosh-director/lib/bosh/director/deployment_plan/manual_network_subnet.rb

```diff
@@ -1,8 +1,10 @@
+require 'netaddr'
+
 module Bosh::Director
   module DeploymentPlan
     class ManualNetworkSubnet < Subnet
       extend ValidationHelper
-      extend IpUtil
+      include IpUtil
```

Comment:
Since visibility of IpUtils is unchanged I wouldn't switch from extend to include. All IpUtil methods are static anyhow so extend should be fine.